### ISSUE TITLE
Automatic report blob (attachment) generation 

### ISSUE THREAD

**Issue by damsleth** [Puzzlepart/did#1094]
- [X] I have searched for existing issues to avoid creating duplicates

### TLDR
Reports should be automatically generated and stored as blobs, available to users with access to them

### Description
Today reports are generated on demand, consuming bandwidth (~30s, ~10MB) and server resources (~50 RU/s, approx 2900 RU total).  
The reports should be made into excel reports on a schedule and made available to users with access, to save on bandwidth and resources.  

In discussing whether to implement this client or server side, a serverside solution will require fewer moving parts and less user interaction.  

A first version will make the Last Year report available as a downloadable excel-document, whereas subsequent versions might include last month, previous confirmed week, last N years, etc.  

Moving parts:  
* Cron job or other timing device on the server
* Server side functionality for retrieving the report
* Service to convert the data to an excel document
* Blob storage component to upload the document to CosmosDB and hook it up to an access level
* UI Component linking to the blob, fetching it and converting it to .xlsx client side
